iidm-ddb/iidm-ddb-ejb/src/main/java/eu/itesla_project/iidm/ddb/service/DDBManagerBean.java

```diff
@@ -538,7 +538,7 @@ public Float getFloatParameter(Equipment equipment,
         ParameterFloat par = getParameterOrDefaultByNameAndType(equipment, simulator,
                 name,
                 ParameterFloat.class);
-        return ((par != null) ? (par.getValue()) : null);
+        return (par != null) ? (par.getValue()) : null;
```

Comment:
@mathbagu 
Should we remove the parentheses around par.getValue()?

Reply:
Yes !